### DISCUSSION TITLE
how do i do this?

### DISCUSSION THREAD

**Discussion by Bink-lab** [givanz/Vvveb#37] [General]
i just installed vvveb from their website but idk how to start it and it doesnt have a editor,html

**Comment by givanz** [comment]
For Vvveb CMS you need to have php installed, instead of http://localhost/editor.html you need to access http://localhost/ directly and install will start.

https://docs.vvveb.com/installation

You can use https://www.wampserver.com/en/ or https://www.apachefriends.org/download.html for localhost or any shared hosting account or server with php support.

**Reply by Bink-lab** [reply]
Thanks!
